.gitignore

```diff
@@ -1 +1,2 @@
-node_modules/
\ No newline at end of file
+node_modules/
+*.html
```

Comment:
This one is slightly risky, maybe worth special-casing a specific directory if some script outputs HTML files there? Unless we're 100% there will never be an HTML file in this repo — and that we don't forget. Otherwise I can see some work never committed to the repo, without anyone noticing.

Reply:
Good point, I can update this to something like `local-testing.html` so it's clear what's being ignored and we don't run into that risk!